src/components/ReportWelcomeText.tsx

```diff
@@ -1,59 +1,65 @@
+function ReportWelcomeText({report, policy}: ReportWelcomeTextProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
+    const [personalDetails] = useOnyx(ONYXKEYS.PERSONAL_DETAILS_LIST);
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(report);
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    const [reportNameValuePairs] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${report?.reportID || -1}`);
+    const isArchivedRoom = ReportUtils.isArchivedRoom(report, reportNameValuePairs);
     const isChatRoom = ReportUtils.isChatRoom(report);
     const isSelfDM = ReportUtils.isSelfDM(report);
     const isInvoiceRoom = ReportUtils.isInvoiceRoom(report);
     const isSystemChat = ReportUtils.isSystemChat(report);
     const isDefault = !(isChatRoom || isPolicyExpenseChat || isSelfDM || isInvoiceRoom || isSystemChat);
-    const participantAccountIDs = ReportUtils.getParticipantsAccountIDsForDisplay(report);
+    const participantAccountIDs = ReportUtils.getParticipantsAccountIDsForDisplay(report, undefined, undefined, true);
     const isMultipleParticipant = participantAccountIDs.length > 1;
     const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(OptionsListUtils.getPersonalDetailsForAccountIDs(participantAccountIDs, personalDetails), isMultipleParticipant);
     const welcomeMessage = SidebarUtils.getWelcomeMessage(report, policy);
     const moneyRequestOptions = ReportUtils.temporary_getMoneyRequestOptions(report, policy, participantAccountIDs);
-    const additionalText = moneyRequestOptions
-        .filter(
-            (item): item is Exclude<IOUType, typeof CONST.IOU.TYPE.REQUEST | typeof CONST.IOU.TYPE.SEND | typeof CONST.IOU.TYPE.CREATE | typeof CONST.IOU.TYPE.INVOICE> =>
-                item !== CONST.IOU.TYPE.INVOICE,
-        )
-        .map((item) => translate(`reportActionsView.iouTypes.${item}`))
+    const filteredOptions = moneyRequestOptions.filter(
+        (item): item is Exclude<IOUType, typeof CONST.IOU.TYPE.REQUEST | typeof CONST.IOU.TYPE.SEND | typeof CONST.IOU.TYPE.CREATE | typeof CONST.IOU.TYPE.INVOICE> =>
+            item !== CONST.IOU.TYPE.INVOICE,
+    );
+    const additionalText = filteredOptions
+        .map((item, index) => `${index === filteredOptions.length - 1 ? `${translate('common.or')} ` : ''}${translate(`reportActionsView.iouTypes.${item}`)}`)
         .join(', ');
     const canEditPolicyDescription = ReportUtils.canEditPolicyDescription(policy);
     const reportName = ReportUtils.getReportName(report);
+    const shouldShowUsePlusButtonText =
+        (moneyRequestOptions.includes(CONST.IOU.TYPE.PAY) ||
+            moneyRequestOptions.includes(CONST.IOU.TYPE.SUBMIT) ||
+            moneyRequestOptions.includes(CONST.IOU.TYPE.TRACK) ||
+            moneyRequestOptions.includes(CONST.IOU.TYPE.SPLIT)) &&
+        !isPolicyExpenseChat;
```

Comment:
@nkdengineer Is there a reason for adding this `!isPolicyExpenseChat` condition?